RecoPixelVertexing/PixelLowPtUtilities/bin/ClusterShapeAnalyzer.cpp

```diff
@@ -13,354 +13,343 @@
+      if (args.verbose)
+        printOut(histo, c, flag);
+    }
+
+    static void printToFile(const TH2F& h2, const std::string& fileName) {
+      std::ofstream file;
+      file.exceptions(std::ofstream::failbit | std::ofstream::badbit);
+      file.open(fileName);
+
+      for (int i = 1; i <= h2.GetNbinsX(); i++) {
+        for (int j = 1; j <= h2.GetNbinsY(); j++)
+          file << " " << h2.GetXaxis()->GetBinLowEdge(i) << " " << h2.GetYaxis()->GetBinLowEdge(j) << " "
+               << h2.GetBinContent(i, j) << "\n";
+
+        file << " " << h2.GetXaxis()->GetBinLowEdge(i) << " " << h2.GetYaxis()->GetXmax() << " 0\n\n";
+      }
+
+      for (int j = 1; j <= h2.GetNbinsY(); j++) {
+        file << " " << h2.GetXaxis()->GetXmax() << " " << h2.GetYaxis()->GetBinLowEdge(j) << " 0\n";
+      }
+
+      file << " " << h2.GetXaxis()->GetXmax() << " " << h2.GetYaxis()->GetXmax() << " 0" << std::endl;
     }
 
-private:
+  private:
     const Arguments args;
-};
-
-} // namespace cluster_shape_filter
-
-int main(int argc, char* argv[])
-{
-    namespace po = boost::program_options;
-    using namespace cluster_shape_filter;
-
-    Arguments args;
-    po::options_description desc("Cluster shape filter analyzer");
-    std::ostringstream ss_loss;
-    ss_loss << std::setprecision(4) << args.loss;
-    desc.add_options()
-            ("help", "print help message")
-            ("input", po::value<std::string>(&args.input)->required(), "input file with extracted cluster shapes")
-            ("output", po::value<std::string>(&args.output)->required(), "output calibration file")
-            ("map-output",  po::value<std::string>(&args.map_output)->default_value(""), "output calibration file")
-            ("loss", po::value<double>(&args.loss)->default_value(args.loss, ss_loss.str()),
-             "(1 - efficiency) threshold for the WP definition")
-            ("min-entries", po::value<size_t>(&args.minEntries)->default_value(args.minEntries), "")
-            ("ex-max", po::value<int>(&args.exMax)->default_value(args.exMax), "")
-            ("ey-max", po::value<int>(&args.eyMax)->default_value(args.eyMax), "")
-            ("verbose", po::bool_switch(&args.verbose));
-
-    try {
-        po::variables_map variables;
-        po::store(po::command_line_parser(argc, argv).options(desc).run(), variables);
-        if(variables.count("help")) {
-            std::cout << desc << std::endl;
-            return 0;
-        }
-        po::notify(variables);
+  };
+
+}  // namespace cluster_shape_filter
+
+int main(int argc, char* argv[]) {
+  namespace po = boost::program_options;
+  using namespace cluster_shape_filter;
 
-        cluster_shape_filter::ClusterAnalyzer analyzer(args);
-        analyzer.analyze();
+  Arguments args;
+  po::options_description desc("Cluster shape filter analyzer");
+  std::ostringstream ss_loss;
+  ss_loss << std::setprecision(4) << args.loss;
+  desc.add_options()("help", "print help message")(
```

Comment:
The automatic reformatting here looks rather weird...

Reply:
indeed... looks like `clang-format` doesn't like `boost::program_options` syntax much